toolbox/fdc3-compliance/src/test/fdc3.broadcast.ts

```diff
@@ -0,0 +1,8 @@
+describe("fdc3.broadcast", async () => {
+  it("Method is callable", async () => {
+    window.fdc3.broadcast({
```

Comment:
Shouldn't we also await the method?

Reply:
Not in 1.2, broadcast becomes async in 2.0